src/Registration.java

```diff
@@ -0,0 +1,36 @@
+public class Registration {
+    private String nickname;
+    private String password;
+    private String email;
+    private String youCatFavouriteToy;
```

Comment:
Pls delete this variable and its getter and setter

Reply:
r u sure, mr?

Reply:
yes pls delete and add sth new

Reply:
ok, i will add more cats

Reply:
whatever i'll let it pass for now